web-marketplace/src/lib/rdf/rdf.unknown-fields.ts

```diff
@@ -65,11 +69,34 @@ const knownProjectFields: KnownProjectFields[] = [
   'regen:cfcProjectPage',
   'regen:offsetProtocol',
   'regen:projectDesignDocument',
+  'regen:ecosystemType',
+
+  // project page only
+  'regen:creditClassId',
+  'regen:previewPhoto',
+  'regen:galleryPhotos',
+  'schema:creditText',
+  'schema:description',
+  'regen:story',
+  'regen:storyTitle',
+  'regen:storyMedia',
+  'schema:image',
+  'regen:glanceText',
+  'regen:landStory',
+  'regen:projectQuote',
+  'regen:boundaries',
+  'regen:landManagementActions',
+
+  // terrasos
+  'regen:marketType',
+  'regen:projectDuration',
+  'regen:managementAreas',
+  'regen:conservationStatus',
+  'regen:ecologicalConnectivityIndex',
+  'regen:socialCulturalIndex',
```

Comment:
@S4mmyb @wgardiner please let me know if I'm missing any terrasos metadata fields that should **not** be shown in the additional info tab

Reply:
these look good to me